### PR TITLE
New bulk file upload

### PR DESCRIPTION
**Pull request by Vedo-Kodecta**

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "@types/uuid": "^9.0.8",
     "eslint": "^8",
```
Comment (ErminMaksumic) [inline react/package.json]: Could you please remove this package?

**Review by ErminMaksumic — CHANGES_REQUESTED** [review]

File: app/Services/ProductService.php [hunk 2]
```diff
@@ -223,13 +226,44 @@ public function generateReportForProductStatesGraph($request)
...
+
+            return response()->json(['message' => 'File upload processing completed']);
+        } catch (\Exception $e) {
+            DB::rollBack();
+
+            // Log the error
+            Log::error("Error processing file upload: " . $e->getMessage());
+
+            // Return a 500 error response
+            return response()->json(['error' => 'File upload processing failed'], 500);
```
Comment (ErminMaksumic) [inline app/Services/ProductService.php]: Could you please implement a logic for removing csv file?

**Review by ErminMaksumic — CHANGES_REQUESTED** [review]

File: react/package.json (hunk 1 above)
Comment (Vedo-Kodecta) [reply react/package.json]: removed, please check again

File: app/Services/ProductService.php (hunk 2 above)
Comment (Vedo-Kodecta) [reply app/Services/ProductService.php]: the logic has been implemented, please double check it again. cheers

**Review by ErminMaksumic — APPROVED** [review]

**Pull request merged**